src/proxy/checksumtable.go

```diff
@@ -22,8 +23,32 @@ func (spanner *Spanner) handleChecksumTable(session *driver.Session, query strin
 	database := session.Schema()
 	ast := node.(*sqlparser.Checksum)
 	table := ast.Table.Name.String()
+	schema := ast.Table.Qualifier.String()
+
+	// Format: db.tbl
+	if schema != "" {
```

Comment:
1. Need review the code again, 
if the schema is "", the table without the schema, but MySQL return with schema;
2. Need add the test about the case.

Reply:
> 1. Need review the code again,
>    if the schema is "", the table without the schema, but MySQL return with schema;
> 2. Need add the test about the case.

Done! Plelease review again.